docs/getting-started/upgrade/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Upgrade Shopify App CLI
+section: getting-started
+toc: false
+redirect_from: "/upgrade/"
+---
+
+You can manage upgrades to Shopify App CLI with the package manager for your platform.
+
+### Brew (MacOS)
+
+```console
+$ brew update && brew upgrade shopify-cli
+```
+
+### apt (Debian, Ubuntu)
+
+```console
+$ sudo apt update
+$ sudo apt upgrade shopify-cli
+```
```

Comment:
Nitpick: Sorry to bring this up again, but should we do the same for brew to make it consistent?

Reply:
It's not needed for `brew` (see the video!)

Reply:
Actually, these two commands assume that `shopify-cli` is a package in a repo, which it's not (yet).  The upgrade will be to download a newer file and do `sudo apt install shopify-cli-x.y.z.deb`, e.g., basically the same as the install command?